lib/Minz/View.php

```diff
@@ -137,11 +140,15 @@ public function helperToString($helper) {
 	}
 
 	/**
-	 * Permet de choisir si on souhaite utiliser le layout
-	 * @param $use true si on souhaite utiliser le layout, false sinon
+	 * Choose the current view layout.
+	 * @param $layout the layout name to use, false to use no layouts.
 	 */
-	public function _useLayout ($use) {
```

Comment:
I believe someone had built an extension doing something similar (custom view) but I cannot immediately find a reference, and I cannot remember which calls were used. Probably this one, in which case this change would break it. Maybe you could keep an additional legacy `_useLayout()` calling `_layout()`

Reply:
good catch! Just updated the PR to bring back a deprecated `_useLayout` function. I also added a `Minz_Log::warning` to warn about the deprecation.